package.json

```diff
@@ -37,6 +37,10 @@
     "@material-ui/lab": "^4.0.0-alpha.60",
     "@material-ui/pickers": "^3.0",
     "@material-ui/styles": "^4.0",
+    "@mui/icons-material": "^5.15.3",
```

Comment:
@MattiaPrimavera Is it necessary to have more relaxed `mui` version constraints?

e.g. accepts all minor versions and their successive patches?

"@mui/icons-material": "^5.0"

Reply:
It is indeed possible @pieeri, but I think we could make use of relaxed dependencies in the development environment, whereas I would probably make sure that we install the dependencies using `yarn.lock` file in the CI, as [yarn docs](https://classic.yarnpkg.com/en/docs/cli/install) state :

```
Running yarn with no command will run yarn install, passing through any provided flags.

If you need reproducible dependencies, which is usually the case with the continuous integration systems, you should pass --frozen-lockfile flag.
```

Reply:
I checked using the [semver simulator](https://semver.npmjs.com/) and the following declarations

```
"@mui/icons-material": "^5.15.3",
```

or

```
"@mui/icons-material": "^5.0"
```

seem to result in the same versions upgrade, the only difference is that the first declaration starts from package 5.15.3 but allows to both get the new minors and patches versions

Reply:
Solved, we stick to `"@mui/icons-material": "^5.15.3"`